src/isar/models/communication/queues/queues.py

```diff
@@ -7,11 +7,26 @@
 
 class Queues:
     def __init__(self) -> None:
-        self.start_mission: QueueIO = QueueIO(input_size=1, output_size=1)
-        self.stop_mission: QueueIO = QueueIO(input_size=1, output_size=1)
-        self.pause_mission: QueueIO = QueueIO(input_size=1, output_size=1)
-        self.resume_mission: QueueIO = QueueIO(input_size=1, output_size=1)
-        self.single_action: QueueIO = QueueIO(input_size=1, output_size=1)
+        self.api_start_mission: QueueIO = QueueIO(input_size=1, output_size=1)
```

Comment:
Could it be worth splitting this into several queues classes so it is clear which threads they facilitate communication between?

Reply:
You mean having the response in a different queue than the request? Currently these are just renamed versions of the already existing queues, but I could make an issue on splitting it up so that the API thread does not listen to the same thread where it sends the request to the state machine.

Reply:
Ok I think I understand what you mean. So that each group of queues should be its own subclass of the queue utility class? I could do that

Reply:
Here you go https://github.com/equinor/isar/issues/720